Iced.UnitTests/Intel/InstructionTests/InstructionTests_Misc.cs

```diff
@@ -54,7 +54,7 @@ public sealed class InstructionTests_Misc {
 		void Code_is_not_too_big() {
 			int maxValue = GetEnumSize(typeof(Code)) - 1;
 			Assert.True(maxValue < (1 << Instruction.TEST_CodeBits));
-			Assert.True(maxValue >= (1 << (Instruction.TEST_CodeBits - 1)));
+			Assert.True(maxValue >= (1 << (Instruction.TEST_CodeBits - 2)));
```

Comment:
You should update CodeBits in Instruction.CodeFlags and its comment.

Reply:
Originally, I did that, but it was weird to have a hole with `RoundingControlShift` (that I didn't change), if that's ok, I can change `CodeFlags.CodeBits` only otherwise do you want to shift everything? (Not sure about the implications)

Reply:
Just the bit count is enough. It will regain its extra bit in the near future when I add undocumented instructions and Intel's deprecated KNC instructions.